starter-code/functions-and-arrays.js

```diff
@@ -20,142 +20,84 @@ var words = [
   "orchard",
   "crackpot"
 ];
+
 var longest = findLongestWord(words);
 console.log(longest);
 
-// Calculating a Sum
-function sumArray (array) {
-
+function findLongestWord(){
+  var result = ""
+  for(var i=0 ; i<words.length;i++){
+    if(words[i].length > result.length) {
+      result = words[i];
+    }
+  }
+  return result
 }
+/*----------------------*/
 
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
-var total = sumArray(numbers);
-console.log(total);
 
-// Calculate the Average
-function averageNumbers (array) {
+function sumArray(array){
+// Your Code here
+var sum=0 ;
+  for(i=0; i<numbers.length;i++){
+    sum = sum+numbers[i];
 
+  }
+console.log(sum);
 }
+var total = sumArray(numbers);
+// 87
 
-var numbers = [2, 6, 9, 10, 7, 4, 1, 9];
-var average = averageNumbers(numbers);
-console.log(average);
-
-// Array of Strings
-function averageWordLength (array) {
-
-}
-
-var words = [
-  "seat",
-  "correspond",
-  "linen",
-  "motif",
-  "hole",
-  "smell",
-  "smart",
-  "chaos",
-  "fuel",
-  "palace"
-];
-var averageLength = averageWordLength(words);
-console.log(averageLength);
-
-// Unique Arrays
-function uniquifyArray (array) {
 
-}
+var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
-var words = [
-  "crab",
-  "poison",
-  "contagious",
-  "simple",
-  "bring",
-  "sharp",
-  "playground",
-  "poison",
-  "communion",
-  "simple",
-  "bring"
-];
-var uniqued = uniquifyArray(words);
-console.log(uniqued);
+function sumArray(array){
```

Comment:
You forgot to change the name of the function. Copy-pasting is treacherous; I suggest people to avoid it.